src/Components/CommentsWidget.tsx

```diff
@@ -208,12 +208,37 @@ export function CommentForm(props: CommentFormProps) {
 
     const target = `${savedLocale}/${document.location.pathname.replace(/^\s*\/*\s*|\s*\/*\s*$/gm, '')}`;
 
+    function calculateMessageSusScore(message: string) {
```

Comment:
I feel like "sus score" might not exactly be an intuitive concept, so a comment that explains the intention would probably be good. :D